lib/action_policy/graphql/authorized_field.rb

```diff
@@ -24,15 +40,15 @@ def after_resolve(value:, context:, object:, **_rest)
           return value if value.nil?
 
           if options[:raise]
```

Comment:
What about removing graphql-specific specific options instead of reconstructing (i.e. `options_for`):

```suggestion
          if options.delete(:raise)
```

Reply:
I've refactored to extract the problematic options to instance variables in the `apply` hook (can't do it in the constructor because the object is frozen after `super`). I think it's a lot better, hope you agree! 😄